FusionIIIT/Fusion/settings/development.py

```diff
@@ -16,6 +16,15 @@
     }
 }
 
+REST_FRAMEWORK = {
```

Comment:
Doesn't this needs to be added in productions settings also?

Reply:
Added in productions settings.